test/integration/eslint/test/next-lint.test.js

```diff
@@ -110,7 +110,9 @@ describe('Next Lint', () => {
       expect(stdout).toContain(
         'We created the .eslintrc.json file for you and included your selected configuration'
       )
-      expect(eslintrcJson).toMatchObject({ extends: 'next/core-web-vitals' })
+      expect(eslintrcJson).toMatchObject({
+        extends: ['next/core-web-vitals', 'next/typescript'],
+      })
```

Comment:
Are we missing any tests here?

For example, the linked issue mentions that it catches floating promises so perhaps we need a test for that?

Reply:
I don't recall ever seeing a project add explicit tests for particular lint rules being enabled. It's an interesting idea though.

Generally, the presence of something like `next/typescript` in the `extends` of an ESLint config has been considered enough. This is essentially a `hasBeenCalledWith` for the ESLint API.

Reply:
I thought there's a CLI command to print the effective config. I'll check if that helps here as a snapshot test.